### PR TITLE
Dev to main [Add fix for invalid date (#900)]

### PR DESCRIPTION
**Pull request by Maheima**
Date: 30-09-2023

Developer Name: @Maheima 

**Dev Tested?**
  Yes
**PR List**
https://github.com/Real-Dev-Squad/website-status/pull/900

**Backend changes**
https://github.com/Real-Dev-Squad/website-backend/pull/1566

**Frontend Changes**
Yes

**Breaking changes (If your feature is breaking/missing something please mention pending tickets)**
None

**Deployment notes**
Any special instructions for the deployment - No

**Tested in staging**
Yes

https://www.loom.com/share/24f227198cd84431a6b606ebef48924a?sid=af687f41-c038-4519-a501-e8e95805b1a4

### CODE REVIEW

**Comment by vinit717** [comment]
Please follow format for dev to main

**Review by Pratiyushkumar — APPROVED** [review]

File: src/components/tasks/card/index.tsx [hunk 1]
```diff
@@ -93,8 +93,17 @@ const Card: FC<CardProps> = ({
...
-    const localStartedOn = new Date(parseInt(cardDetails.startedOn, 10) * 1000);
-    const fromNowStartedOn = moment(localStartedOn).fromNow();
+    function getStartedAgo() {
+        if (cardDetails.startedOn === undefined) {
+            return 'N/A';
+        } else {
+            const localStartedOn = new Date(
+                parseInt(cardDetails.startedOn, 10) * 1000
+            );
+            const fromNowStartedOn = moment(localStartedOn).fromNow();
```
Comment (iamitprakash) [inline src/components/tasks/card/index.tsx:103]: i do see issues here @Maheima

**Comment by Maheima** [comment]
> Please follow format for dev to main

Please check @vinit717

File: src/components/tasks/card/index.tsx (hunk 1 above)
Comment (Maheima) [reply src/components/tasks/card/index.tsx:103]: where?

**Comment by vinit717** [comment]
> > Please follow format for dev to main
> 
> Please check @vinit717

This is pr format not dev to main sync format

**Comment by Maheima** [comment]
> > > Please follow format for dev to main
> > 
> > 
> > Please check @vinit717
> 
> This is pr format not dev to main sync format

https://github.com/Real-Dev-Squad/website-status/pull/895 this is the one I found. can you point me to the right format?

File: src/components/tasks/card/index.tsx (hunk 1 above)
Comment (iamitprakash) [reply src/components/tasks/card/index.tsx:103]: localStartedOn, in which time zone format we are saving date, to TUC, IST or ....

**Review by iamitprakash — APPROVED** [review]

**Review by iamitprakash — CHANGES_REQUESTED** [review]
This branch is out-of-date with the base branch

**Comment by vinit717** [comment]
> > > > Please follow format for dev to main
> > > 
> > > 
> > > Please check @vinit717
> > 
> > 
> > This is pr format not dev to main sync format
> 
> #895 this is the one I found. can you point me to the right format?

add issue link also and the video is local not staging

**Pull request closed without merging**